_locales/_untranslated_en.json

```diff
@@ -55,5 +55,26 @@
   },
   "ask_send_file_desktop":{
     "message": "Send %s to %s?"
+  },
+  "confirm_leave_group": {
+    "message": "Leave"
+  },
+  "confirm_delete_chat": {
+    "message": "Delete"
+  },
+  "confirm_block_contact": {
+    "message": "Block"
+  },
+  "confirm_unblock_contact": {
+    "message": "Unblock"
+  },
+  "confirm_forget_login": {
```

Comment:
please just use "forget" to create this new string. this way, we can use strings more widely; in general, we're create very specific labels only if we really realize there is a need. alternatively, we have delete_account, delete_account_ask which are used on ios. however, we can streamline things later, this should be no blocker.

Reply:
I can Imagine that the meaning can get lost when just using an English string. Languages are more complicated that key-value, words can have different meanings in different contexts. So I would be careful with universal translating single English word and using it at multiple places.